### DISCUSSION TITLE
Is there a way to do count or exists?

### DISCUSSION THREAD

**Discussion by alper** [SeaQL/sea-orm#484] [Q&A] [answered]
At the moment I can't find a count method in the ORM at all so I just do the query and count the length of the Vector? That can't be a good way.

I would like to know whether a record exists in the database? 

I'd like to do something like `get_or_create`.

**Comment by billy1624** [comment]
Hi again loll, yes, you can use the `count` method on any selector, see https://docs.rs/sea-orm/0.5.0/sea_orm/entity/prelude/trait.PaginatorTrait.html#method.count

**Comment by vrozhdestvenskyi** [comment]
Hi all!
> I'd like to do something like get_or_create.

Any suggestions on how to implement the `get_or_create` method?

**Comment by Expurple** [comment, marked as answer]
`exists` has been merged in https://github.com/SeaQL/sea-orm/pull/2623
